cmd/kubemark/hollow-node.go

```diff
@@ -70,13 +72,39 @@ func makeTempDirOrDie(prefix string, baseDir string) string {
 	return tempDir
 }
 
+func createClientFromFile(path string) (*client.Client, error) {
+	c, err := clientcmd.LoadFromFile(path)
+	if err != nil {
+		return nil, fmt.Errorf("Error while loading kubeconfig from file %v: %v", path, err)
+	}
+	config, err := clientcmd.NewDefaultClientConfig(*c, &clientcmd.ConfigOverrides{}).ClientConfig()
+	if err != nil {
+		return nil, fmt.Errorf("Error while creating kubeconfig: %v", err)
+	}
+	client, err := client.New(config)
+	if err != nil {
+		return nil, fmt.Errorf("Error while creating client: %v", err)
+	}
+	if client.Timeout == 0 {
+		client.Timeout = 30 * time.Second
+	}
+	return client, nil
+}
+
 func main() {
 	runtime.GOMAXPROCS(runtime.NumCPU())
 	addFlags(pflag.CommandLine)
 	util.InitFlags()
 
+	// If kubeconfigPath is not set, try default
+	if kubeconfigPath == "" {
```

Comment:
Why not just use /kubeconfig/kubeconfig as the default value for the flag? It seems like you are reimplementing flag defaults doing it this way.